L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorSuperTriggerCellImpl.h

```diff
@@ -22,14 +22,16 @@ class HGCalConcentratorSuperTriggerCellImpl
   private:
 
     int getSuperTriggerCellId(int detid) const ;
+    static const int split = 0x3a;
```

Comment:
Class members should have trailing underscores. Also to make clear that these variables are constants, they can be prefixed with `k` (e.g. `kSplit_`)